### ISSUE TITLE
is Orphanet:595098  equiv to MONDO_0010979 'Timothy syndrome'?

### ISSUE THREAD

**Issue by nicolevasilevsky** [monarch-initiative/mondo#4746]
Related to #4579, this is for the ORDO 4.0 sync.

@sabrinatoro I think this term is equiv to MONDO_0010979 'Timothy syndrome'. Could you review and let me know if you agree? Note, the affected genes are the same in both terms. I didn't check if this exists in OMIM.
````
[Term]
id: MONDO:0035678
name: Timothy syndrome type 1
subset: ordo_subtype_of_a_disorder
synonym: "LQT8 type 1" EXACT  [Orphanet:595098]
synonym: "TS1" EXACT  [Orphanet:595098]
xref: ICD10:I49.8 {source="Orphanet:595098", source="MONDO:MONDO:mondoIsNarrowerThanSource"}
is_a: MONDO:8000031 {source="Orphanet:595098"} ! obsolete subtype of a disorder
relationship: disease_has_basis_in_dysfunction_of http://identifiers.org/hgnc/1390
xref: Orphanet:595098 {source="MONDO:equivalentTo"} ! timothy syndrome type 1
```

**Comment by nicolevasilevsky** [comment]
Note, Orphanet also has these two terms as well that we should add. We need to investigate the relationship of these to the existing Mondo terms. In Mondo we have:
-'Timothy syndrome'
--'Timothy syndrome type 1 (disorder)'
--'Timothy syndrome type 2 (disorder)'

```
[Term]
id: MONDO:0035679
name: timothy syndrome type 2
subset: ordo_subtype_of_a_disorder
synonym: "LQT8 type 2" EXACT  [Orphanet:595105]
synonym: "TS2" EXACT  [Orphanet:595105]
xref: ICD10:I49.8 {source="Orphanet:595105"} {source="MONDO:MONDO:mondoIsNarrowerThanSource"}
is_a: MONDO:8000031 {source="Orphanet:595105"} ! obsolete subtype of a disorder
xref: Orphanet:595105 {source="MONDO:equivalentTo"} ! timothy syndrome type 2

[Term]
id: MONDO:0035680
name: atypical timothy syndrome
subset: ordo_subtype_of_a_disorder
synonym: "ATS" EXACT  [Orphanet:595109]
synonym: "Atypical LQT8" EXACT  [Orphanet:595109]
xref: ICD10:I49.8 {source="Orphanet:595109"} {source="MONDO:MONDO:mondoIsNarrowerThanSource"}
is_a: MONDO:8000031 {source="Orphanet:595109"} ! obsolete subtype of a disorder
xref: Orphanet:595109 {source="MONDO:equivalentTo"} ! atypical timothy syndrome
```

**Comment by sabrinatoro** [comment]
This is a bit complicated... 
Orphanet:65283 = Timothy syndrome (MONDO:0010979)
Definition: (...) There are three clinical phenotypes recognized, the **classical types** that present with a prolonged QT interval and either with **(TS1)** or without **(TS2)** cutaneous syndactyly of fingers and toes. The **atypical form (ATS)** causes multi-system health concerns but not necessarily with prolonged QT.

In Mondo: 
Timothy syndrome (MONDO:0010979) - Orphanet:65283
–Timothy syndrome type 1 (=classic form) = MONDO:0021171
–Timothy syndrome type 2 (=atypical form) = MONDO:0021172

So I think that the best way forward is :
1) rename Mondo terms: 
   MONDO:0021171 should be renamed "classical timothy syndrome"
   MONDO:0021172 should be renamed "atypical timothy syndrome"
2) Orphanet:595109 : should be x-ref of  MONDO:0021172  (ie MONDO:0035680 should be merged to MONDO:0021172)
3) Orphanet:595105 (MONDO:0035679; timothy syndrome type 2) should be a child of MONDO:0021171
4) Orphanet:595098 (MONDO:0035678; Timothy syndrome type 1) should be a child of MONDO:0021171

Note: ~type 1 and ~type 2 might bring confusion... Maybe we could rename to "classical timothy syndrome with cutaneous syndactyly" and "classical timothy syndrome without cutaneous syndactyly" 
 It is a long name, but it might be better to avoid confusion. 

(@nicolevasilevsky )

**Comment by nicolevasilevsky** [comment]
agree!
